apps/gjenlevendepensjon-ui/.eslintrc.yml

```diff
@@ -34,7 +34,8 @@ rules: {
      '@typescript-eslint/no-empty-interface': 'off',
      '@typescript-eslint/no-extra-non-null-assertion': 'off',
      '@typescript-eslint/ban-types': 'off',
-     '@typescript-eslint/no-non-null-assertion': 'off'
+     '@typescript-eslint/no-non-null-assertion': 'off',
+     'react/no-children-prop': 'off'
```

Comment:
Ville kanskje ikke fjernet denne. Er der for å unngå at man gjør en blemme med children. 

Si du har noe à la `<CheckboxGroup children={checkboxes}>{noeInnholdHer}</CheckboxGroup>`... 
I dette tilfellet vil `props.children` være `{noeInnholdHer}` og man mister listen med checkboxes som er sendt inn.